### ISSUE TITLE
0002234:
Confirm password dialog has no edit box if password contains forbidden characters

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#1198]
**Reported by xilix on 4 Feb 2010 08:23**

**Version:** Mialena (2010-03) Milestone 1

When I add a new user and enter a non valid password, the &#039;confirm password&#039; dialog has no edit box. If I click on OK, the password field is marked red.
But Tine 2.0 does not tell me that the password contains not valid characters (ö,ä,...). It just tells, that the two passwords are not equal.

**Comment by pschuele** [comment]
i agree, the confirm pw dlg/widget should be improved - the usability is bad ...
